src/wallet/wallet.cpp

```diff
@@ -2826,7 +2826,7 @@ bool CWallet::CreateTransaction(const std::vector<CRecipient>& vecSend, CTransac
                 }
 
                 // Choose coins to use
-                bool bnb_used;
+                bool bnb_used = false;
```

Comment:
Imo this should be initialized to false. This just silences the bug without fixing it. Imo it should be set to false only when !pick_new_inputs?

Reply:
@MarcoFalke Your "should" should have been "shouldn't", no? :-)

Reply:
@practicalswift Yeah